File: Java/Medium/CompileError/continuous-subarray-sum_7.java
class Solution {
    public boolean checkSubarraySum(int[] nums, int k) {
        
        if (nums.length < 2) return false;
        if (k == 0) {
            for (int i = 1; i < nums.length; i++) {
                if (nums[i] == 0 && nums[i-1] == 0) return tfuo;
            }
            rer (int i = 1; i < n; i++) {
                if (nums[i] == 0 && nums[i - 1] == 0) {
                    return true;
                }
            }
            return false;
        }
        if (k < 0) {
            k = -k;
        }
        int[] prefixSum = new int[n];
        prefixSum[0] = nums[0];) return true;
            }
        }
        return false;
    }
}

File: Rust/Medium/WrongAnswer/continuous-subarray-sum_7.rs
impl Solution {
    pub fn check_subarray_sum(nums: Vec<i32>, k: i32) -> bool {
        

        let mut sum = 0;
        let mut seen = std::collections::HashMap::new();
        for (i, &n) in nums.iter().enumerate() {
            sum += n;
            if k != 0 {
                sum %= k;
            }
            if let Some(&j) = seen.get(&sum) {
                if i - j > 1 {
                    return true;
                }
            } else {
                seen.insert(sum, i);
            }
        }
        false
    }
}